### PR TITLE
feat: initial schema for query consumer service

### PR DESCRIPTION
**Pull request by arunbakt**
Initial schema for the web-map query consumer. I have omitted the tables for `tree` and `region` tables for now.

### CODE REVIEW

File: database/migrations/sqls/20210324215801-raw-capture-cluster-up.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE raw_capture_cluster
+(
+	id serial NOT NULL PRIMARY KEY,
```
Comment (deepwinter) [inline database/migrations/sqls/20210324215801-raw-capture-cluster-up.sql:3]: I think these should be NOT NULL

File: database/migrations/sqls/20210324215818-capture-cluster-up.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE capture_cluster
+(
+	id serial NOT NULL PRIMARY KEY,
+	count int4 NULL,
```
Comment (deepwinter) [inline database/migrations/sqls/20210324215818-capture-cluster-up.sql:4]: I think these should be NOT NULL

File: database/migrations/sqls/20210326224122-capture-map-up.sql [hunk 3]
```diff
@@ -0,0 +1,30 @@
+CREATE TABLE capture_map
```
Comment (deepwinter) [inline database/migrations/sqls/20210326224122-capture-map-up.sql:1]: let's do "capture_feature"

File: database/migrations/sqls/20210324052748-raw-capture-map-up.sql [hunk 4]
```diff
@@ -0,0 +1,28 @@
+CREATE TABLE raw_capture_map
+(
+    id uuid NOT NULL PRIMARY KEY,
+    image_url varchar NOT NULL,
+    lat numeric NOT NULL,
+    lon numeric NOT NULL,
+    point_geometry geometry(POINT, 4326) NOT NULL,
```
Comment (deepwinter) [inline database/migrations/sqls/20210324052748-raw-capture-map-up.sql:7]: we could do 'location' for this if you like as well

File: database/migrations/sqls/20210326224122-capture-map-up.sql [hunk 5]
```diff
@@ -0,0 +1,30 @@
+CREATE TABLE capture_map
+(
+    id uuid NOT NULL PRIMARY KEY,
+    image_url varchar NOT NULL,
```
Comment (deepwinter) [inline database/migrations/sqls/20210326224122-capture-map-up.sql:4]: do we need the image urls for capture and field user?  would we every query on these?

**Review by deepwinter — CHANGES_REQUESTED** [review]

File: database/migrations/sqls/20210324215801-raw-capture-cluster-up.sql (hunk 1 above)
Comment (arunbakt) [reply database/migrations/sqls/20210324215801-raw-capture-cluster-up.sql:3]: you mean the count should be not null?

File: database/migrations/sqls/20210326224122-capture-map-up.sql (hunk 5 above)
Comment (arunbakt) [reply database/migrations/sqls/20210326224122-capture-map-up.sql:4]: Don;t we need these if the web-map service wants to show the picture?

**Pull request merged**